### PR TITLE
tentative restructring and clarifications around annotations

### LINKED ISSUE #100: Events and Annotations
There are many annotations that one could attach to an event:
* language
* languageSource (enumeration)
* scene information (free form text)
* plot significance (enumeration)
* on/off screen (enumeration)
* type of event (on screen, dialog, description, ...)
* emotion (probably enumeration)
* ...

We have several tools at our disposal:
* a new attribute in the daptm namespace
* a ttm:desc element with daptm:descType attribute.
* a metadata element
* attributes/elements in foreign namespaces
* ...

For the values in the elements or in the attributes, we have the choice between:
* single value or multiple values (space separated, comma separated, ...)
* each value being: a general string, from an enumeration in some registry, or from an enumeration in this spec

I wonder if we shouldn't have a section "Script Event Annotations", with an introduction describing the general design principles and then sub sections for each annotation.

### PR DESCRIPTION
**Pull request by cconcolato**
close #100

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/dapt/pull/161.html" title="Last updated on Jun 14, 2023, 9:01 PM UTC (fc75c67)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/dapt/161/3b9e751...fc75c67.html" title="Last updated on Jun 14, 2023, 9:01 PM UTC (fc75c67)">Diff</a>

### CODE REVIEW

**Comment by cconcolato** [comment]
I'm also wondering if onScreen should not be one of the keyword annotations.

File: index.html [hunk 1]
```diff
@@ -900,91 +900,112 @@ <h4>Text Language Source</h4>
...
-        </section>
-        <section>
-          <h4>Script Event Type</h4>
-          <p>The <dfn>Script Event Type</dfn> property provides one or more space-separated keywords representing the type of the <a>Script Event</a>,
-            i.e. spoken text, or on-screen text,
-            and in the latter case, the type of on-screen text (title, credit, location, ...).
-            The possible keywords are indicated in the registry at XXXX.</p>
-          <p class="ednote">Registry to be defined.</p>
-          <p>The <a>Script Event Type</a> is represented in a <a>DAPT Document</a> by the following attribute:
+            <p>The <code>&lt;ttm:desc&gt;</code> element MAY have a <code>daptm:descType</code> attribute specified to indicate the type of description. The <code>daptm:descType</code> attribute is defined below. Its possible values are as indicated in the registry at YYY.</p>
```
Comment (nigelmegitt) [inline index.html:967]: Passing thought: we really should have `descType` in TTML. Sigh.

File: index.html [hunk 2]
```diff
@@ -900,91 +900,112 @@ <h4>Text Language Source</h4>
...
-     daptm:eventType=&quot;dialogue&quot;
-...
-&lt;/div&gt;
-...
+            <pre class="example"
+              data-include="examples/event-desc-descType.xml"
+              data-include-format="text">
             </pre>
-          </p>
+            <p>Multiple <code>&lt;ttm:desc&gt;</code> elements MAY be present with different values of <code>daptm:descType</code>, as in the following example.</p>
```
Comment (nigelmegitt) [inline index.html:978]: Thought in passing (maybe a separate issue): do we want to constrain that no more than one `ttm:desc` element with the same parent element has the same `daptm:descType` attribute value?

**Review by nigelmegitt — COMMENTED** [review]
Questions and suggestions.

**Comment by cconcolato** [comment]
Dropping this PR as overtaken by events and not critical for CR.

**Pull request closed without merging**